css/styles.css

```diff
@@ -278,3 +282,104 @@ footer > .inner {
     }
 }
 
+@media (max-width: 760px) {
+    .wrapper > *,
+    header > *,
+    footer > * {
+        max-width: 690px;
```

Comment:
You created intermediate breakpoint for the layout. It's nice, because layout has no hard transition between 760 and 425. But you forget to do rules for range from 600 to 690. So in the range behavior of the layout is responsive instead of adaptive.